quinn/src/lib.rs

```diff
@@ -96,6 +96,7 @@ enum ConnectionEvent {
     },
     Proto(proto::ConnectionEvent),
     Ping,
+    UpdateRemoteConnectionId,
```

Comment:
It looks like these two are only ever sent in sequence. Maybe we should have a single more abstract event, e.g. `Rebind` or `LocalAddressChanged`?

Reply:
I've merged the two into LocalAddressChanged, and force-pushed a new version of the PR.  The commit message  references a newly created issue about the server-side case.  Thanks.